src/layout/constants.tsx

```diff
@@ -1,37 +1,116 @@
 import React from 'react';
-import { HomePage, FavoritePage, UserPage, LoginPage, PageNotFound } from '../pages'
-import { RouteConfigurationType } from '../types';
-import { Pages } from '../enum';
+import {
+  HomePage,
+  FavoritePage,
+  HistoryPage,
+  LoginPage,
+  PageNotFound,
+  FriendsPage,
+  GroupsPage,
+  MessagePage,
+  NotificationsPage,
+  ProfilePage,
+  VideoPage,
+  UserPage
+} from '../pages'
+import { Pages, PERMISSIONS } from '../enum';
 
-export const routeConfiguration: RouteConfigurationType[] = [
-  {
-    id: Pages.login,
-    isRequireAuth: false,
-    url: '/',
-    component: <LoginPage />,
+
+export const groupPermission = {
+  READ_WRITE: [PERMISSIONS.READ, PERMISSIONS.WRITE],
+  EDIT_UPDATE: [PERMISSIONS.EDIT, PERMISSIONS.UPDATE],
```

Comment:
Here we can apply the same thing - edit and update are practically the same thing.